web-ui/angular.json

```diff
@@ -0,0 +1,132 @@
+{
```

Comment:
is this renaming intended?

Reply:
Yes. In Angular 6, the filename is now `angular.json` instead of `.angular.json`.